### ISSUE TITLE
Wrist Sign

### ISSUE THREAD

**Issue by samardeep331** [obophenotype/human-phenotype-ontology#11207]
**Preferred term label:**
Wrist Sign

**Synonyms**
Wrist Sign (Ghent Criteria)

**Definition (free text, please give PubMed ID)**
The wrist sign is positive when the tip of the thumb covers the entire fingernail of the fifth finger when wrapped around the contralateral wrist.

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Marfan Syndrome

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added, thanks! HP:0025091
